deploy/helm/sumologic/values.yaml

```diff
@@ -27,6 +27,7 @@ sumologic:
   ## Sumo API endpoint; Leave blank for automatic endpoint discovery and redirection
   ## ref: https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
   endpoint: ""
+  open_endpoint: ""
```

Comment:
It would be good to add new parameters also to [this](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md) documentation.

Reply:
I don't see new parameters added to values.yaml in documentation (https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md). They maybe added in new pull request if you prefer that way.

Reply:
Added all new `values.yaml` properties to README https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1697/commits/b0da9e5fa2ad711ba1942850c4a3085eafdf0d16